lib/OpenQA/Utils.pm

```diff
@@ -513,5 +514,26 @@ sub human_readable_size {
     return $p . _round_a_bit($size) . "GiB";
 }
 
+# query group parents and job groups and let the database sort it for us - and merge it afterwards
+sub job_groups_and_parents {
```

Comment:
actually I would love to move this to a ResultSet class, but I can't decided which one :)